### ISSUE TITLE
Minor: Creates overriding makepkg.conf in user directory

### ISSUE THREAD

**Issue by CJ1169** [vinifmor/bauh#30]
See this link for more information: https://forum.manjaro.org/t/makepkg-ignoring-pkgext-in-makepkg-conf/109338/8
TL;DR Does bauh _have_ to create its own makepkg.conf in the user directory?

**Comment by vinifmor** [comment]
If you have enabled AUR , it will create an optimized version of makepkg in your home folder. This behavior can be disabled via the environment variables **BAUH_ARCH_OPTIMIZE=0**. This is described in the project README file in the AUR (arch) gem section.

**Comment by CJ1169** [comment]
> 
> 
> If you have enabled AUR , it will create an optimized version of makepkg in your home folder. This behavior can be disabled via the environment variable **BAUH_ARCH_OPTIMIZE=0**. This is described in the project README file in the AUR (arch) gem section.

Good to hear the behavior can be modified. However, is it best to have that as the default behavior? I believe it's possible to call makepkg using a custom makepkg.conf that won't be picked up regularly. yay, the aur helper does this, I believe (although not by default). 
i.e. maybe in a ~/.config/bauh directory, a makepkg.conf file could be used and then makepkg directed to use that instead.

Also good work my dude!

**Comment by vinifmor** [comment]
I added this behavior with new Linux users in mind. Non-tech people that does not know anything about compilation and probably won't like some AUR applications taking too long to install. bauh just make a copy of your **/etc/makepkg.conf** in your home folder and add these two lines ( if you haven't defined a value for them ):

**MAKEFLAGS="-j$(nproc)"
COMPRESSXZ=(xz -c -z - --threads=0)**

In the next future, I will add a settings window where you can see all bauh options and enable / disable them.

About informing a custom makepkg, I agree with you that is a better way and was the first thing that came in mind when I was coding. Unfortunately, I didn't find any other way at the time. If you know how to do it, please let me know, and I will change it for the next release.

Cheers

**Comment by CJ1169** [comment]
> I added this behavior with new Linux users in mind. Non-tech people that don't know anything about compilation and probably won't like some AUR applications taking too long to install. bauh just make a copy of your **/etc/makepkg.conf** in your home folder and add these two lines ( if you haven't defined a value for them ):
> 
> **MAKEFLAGS="-j$(nproc)" COMPRESSXZ=(xz -c -z - --threads=0)**
> 
> In the next future, I will add a settings window where you can see all bauh options and enable / disable them.
> 
> About informing a custom makepkg, I agree with you that is a better way and was the first thing that came in mind when I was coding. Unfortunately, I didn't find any other way at the time. If you know how to do it, please let me know, and I will change it for the next release.
> 
> Cheers

Makes sense!
As for passing a custom config to makepkg, I believe 
exec:
```
	if config.MakepkgConf != "" {
		args = append(args, "--config", config.MakepkgConf)
	}
```
config:
```
		MakepkgConf:        "",
```
Does it in Go. This was taken from [yay](https://github.com/Jguer/yay)'s source code. 
TL;DR: passing --config "/etc/makepkg.conf" should do it?
Goodluck! :)

**Comment by vinifmor** [comment]
@CJ1169, sorry for the late response and thank you for your help. I will change the behavior this week and release **0.7.3** with it.

**Comment by vinifmor** [comment]
Already done. If you want to have a look in the code changes -> https://github.com/vinifmor/bauh/commit/374c6971969ea44b4a48495a1ec6eecdf660c75d

In the end of the day I will be publishing it to **AUR** in **bauh-staging** package.

**Comment by CJ1169** [comment]
Good to hear! Lovely work
